README.md

```diff
@@ -1,73 +1,130 @@
-# README
-
-
-# KitchenSync+
---- 
-
-### Description
-
-KitchenSync+ is an online application that can be used to organize groups and shopping lists. Users will register via Google Oauth and be able to create groups called “Communities” that users will be able to search recipes and create shopping lists based on what is required. Individual members of the community will have the ability to self assign needed ingredients that they will purchase for the recipe. Our goal is to make your Kitchen in sync!
-
-### Screen Shots
-
-
-### Developer Notes
-* Ruby version
-  - `2.7.4`
-  
-* Rails version 
-  -`5.2.8` 
-  
-* Implemented Gems 
-  - `Faraday`
-    - [Faraday details](https://lostisland.github.io/faraday/)  
-  - `rspec-rails`
-    - [RSpec-Rails details](https://rspec.info/)
-  - 'launchy'
-    - [Launchy details](https://github.com/copiousfreetime/launchy)
-  - `pry` 
-    - [Pry details](https://rubygems.org/gems/pry/versions/0.14.1)
-  - 'simplcov' 
-    - [Simplecov details](https://github.com/simplecov-ruby/simplecov)
-  - `shoulda-matchers`
-    - [Shoulda-Matchers details](https://github.com/thoughtbot/shoulda-matchers) 
-  - `orderly` 
-    - [Orderly details](https://github.com/jmondo/orderly)
-    
-  - `bootstrap` 
-    - [Bootstrap details](https://github.com/twbs/bootstrap-rubygem)
-  - `rubocop`
-    -[Rubocop details](https://github.com/rubocop/rubocop)
-  
+<div id="header" align="center">
+
+<img width="762" alt="Roe'd To Aid Logo" src="https://images.prismic.io/carbmanager/68bd7d46-5f50-413c-97fe-e37b1c816159_Header-LowFODMAP.jpg?auto=compress%2Cformat&w=0.5&auto=format&fit=max">
+</div>
+
+<h1 align="center"> Welcome to KitchenSync+</h1>
+<h2 align="center">Helping you synchronize meal planning </h2>
+
+
+
+## Table of Contents
+
+- [Project Overview](#project-overview)
+- [Learning Goals](#learning-goals)
```

Comment:
Do you see how this is working? In markdown when you create a link like this, Markdown is "smart" enough to make the link clickable, and it will find the `## Project Overview` on line 22. Kinda neat, yeah?!